### PR TITLE
[HYDRA-485] [HYDRA-544] Bugfix release/ui improve widget jsons

### PR DESCRIPTION
**Pull request by elfenheart**
- Update description for Join fields
- Sort dropdown alphabetically

JIRAs:
- https://issues.cask.co/browse/HYDRA-485
- https://issues.cask.co/browse/HYDRA-544

### CODE REVIEW

File: transform-plugins/widgets/CSVFormatter-transform.json [hunk 1]
```diff
@@ -27,19 +27,19 @@
           "name": "delimiter",
           "widget-attributes": {
             "values": [
+              "CARROT",
```
Comment (sreevatsanraman) [inline transform-plugins/widgets/CSVFormatter-transform.json]: :)

Comment (sreevatsanraman) [inline transform-plugins/widgets/CSVFormatter-transform.json]: Did you mean Caret? https://en.wikipedia.org/wiki/Caret

**Comment by sreevatsanraman** [comment]
One minor comment. Fix and merge

File: transform-plugins/widgets/CSVFormatter-transform.json (hunk 1 above)
Comment (anwar6953) [inline transform-plugins/widgets/CSVFormatter-transform.json]: Seems like it was existing code, from https://github.com/caskdata/hydrator-plugins/pull/32.

In fixing this, you should also make a change here:
https://github.com/caskdata/hydrator-plugins/blob/release/1.4/transform-plugins/src/main/java/co/cask/hydrator/plugin/CSVFormatter.java#L90

**Comment by anwar6953** [comment]
last commit lgtm

**Pull request merged**